igdiscover/init.py

```diff
@@ -29,6 +29,8 @@
 def add_arguments(parser):
 	parser.add_argument('--database', '--db', metavar='PATH', default=None,
 		help='Directory with V.fasta, D.fasta and J.fasta files. If not given, a dialog is shown.')
+	parser.add_argument('--species', '--sp', metavar='SPECIES', default='any',
```

Comment:
You should use `choices=('any', 'human')`. Then the available choices will also be in the `--help` output and there will be an error if anything else is specified.

Reply:
I added choices, also I usually used ArgumentDefaultsHelpFormatter which is nice as it automatically displays defaults, but maybe there is something against it and would be too much of changing.

Reply:
I haven’t used ArgumentDefaultsHelpFormatter, sounds like a good idea. (But that needs to be done separately, not here.)